src/generators/clientFileGenerator.ts

```diff
@@ -94,6 +94,10 @@ export function generateClient(clientDetails: ClientDetails, project: Project) {
       namespaceImport: "coreClient",
       moduleSpecifier: "@azure/core-client"
     });
+    clientFile.addImportDeclaration({
```

Comment:
Do we want to check for the compat flag before adding the import?

Reply:
The check is not necessary. If the import is not used, it will be removed at the end by the prettier